cmd/tellor/main.go

```diff
@@ -172,41 +181,38 @@ func stakeCmd(logSetup func(string) log.Logger, logLevel *string) func(*cli.Cmd)
 	}
 }
 
-func simpleCmd(f func(context.Context, log.Logger) error, logSetup func(string) log.Logger, logLevel *string) func(*cli.Cmd) {
+func simpleCmd(f func(context.Context, log.Logger, rpc.ETHClient, tellorCommon.Contract, tellorCommon.Account) error, logSetup func(string) log.Logger, logLevel *string) func(*cli.Cmd) {
 	return func(cmd *cli.Cmd) {
 		cmd.Action = func() {
-			ExitOnError(f(ctx, logSetup(*logLevel)), "")
+			ExitOnError(f(ctx, logSetup(*logLevel), clt, cont, acc), "")
 		}
 	}
 }
 
-func moveCmd(f func(context.Context, log.Logger, common.Address, *big.Int) error, logSetup func(string) log.Logger, logLevel *string) func(*cli.Cmd) {
+func moveCmd(f func(context.Context, log.Logger, rpc.ETHClient, tellorCommon.Contract, tellorCommon.Account, common.Address, *big.Int) error, logSetup func(string) log.Logger, logLevel *string) func(*cli.Cmd) {
```

Comment:
```suggestion
func moveCmd(
	f func(
		context.Context,
		log.Logger,
		rpc.ETHClient,
		tellorCommon.Contract,
		tellorCommon.Account,
		common.Address, *big.Int,
	) error,
	logSetup func(string) log.Logger,
	logLevel *string,
) func(*cli.Cmd)
```

We will probably refactor this later, but for now can add new lines to make it a little bit more readable